tests/unit/info.tcl

```diff
@@ -465,3 +465,51 @@ start_server {tags {"info" "external:skip"}} {
         }
     }
 }
+
+proc getMemoryStatsProperty {stats property} {
+    set res 0
+    set list_stats [split $stats " "]
+    for {set j 0} {$j < [llength $list_stats]} {incr j} {
+        if {[string match -nocase $property [lindex $list_stats $j]]} {
+            set res [lindex $list_stats [expr $j+1]]
+            break
+        }
+    }
+    return $res
+}
+
+start_server {tags {"info" "external:skip"}} {
+    test {memory: database and pubsub overhead and rehashing dict count} {
+        r flushall
+        set info_mem [r info memory]
+        set mem_stats [r memory stats]
+        assert_equal [getInfoProperty $info_mem mem_overhead_hashtable_rehashing] {0}
+        assert_equal [getMemoryStatsProperty $mem_stats overhead.hashtable.lut] {0}
+        assert_equal [getMemoryStatsProperty $mem_stats overhead.hashtable.rehashing] {0}
+        assert_equal [getMemoryStatsProperty $mem_stats dict.rehashing.count] {0}
```

Comment:
i'm not certain this is needed.

Reply:
It is better to verify all the metrics added, right?